src/licensedcode/data/licenses/nokia.LICENSE

```diff
@@ -0,0 +1,134 @@
+ 1. DEFINITIONS.
```

Comment:
we already have nokos-1.0a for tis. This should be a new rule for nokos-1.0a instead of a new license record